### ISSUE TITLE
Notification - WCAG rule 2.3.3

### ISSUE THREAD

**Issue by SeamusLeonardHPE** [grommet/grommet#7609]
**Failed rule**

2.3.3
Animation from Interactions
Level AAA

Motion animation triggered by interaction can be disabled, unless the animation is essential to the functionality or the information being conveyed.

---

**Notes**

This is a Level AAA fail

The API for this component doesn't give implementers a way to change or turn off animation.
Animation for toast need to check for "prefers-reduced-motion" to pass

**Comment by jcfilben** [comment]
Switching from bug to enhancement since this is a AAA rule
